src/toolkitAPI.c

```diff
@@ -977,12 +977,66 @@ int DLLEXPORT swmm_getSubcatchResult(int index, int type, double *result)
     return(errcode);
 }
 
+int DLLEXPORT swmm_getSubcatchPollut(int index, int type, double **PollutArray)
```

Comment:
Would be useful here to also return the size of the PollutArray being passed out. This will allow for conversion of the array to a list in the python swig wrapper.

Reply:
@michaeltryby, that’s a good comment. If you lightly scroll up and down, can you see if this suggestion should be applied anywhere else? We could kill two stones with one bird ;-)

Reply:
@bemcdonnell Or just lump it in with the other changes as we swig wrap the API.

Reply:
@bemcdonnell @michaeltryby so just to be clear, I should leave `swmm_getSubcatchPollut` as it is for this PR, and I'll update it as we change over to SWIG?